.github/workflows/backend-lint.yml

```diff
@@ -88,5 +88,14 @@ jobs:
           #      `getconf ARG_MAX` - max # bytes of args + environ for exec()
           pre-commit run --files ${{ steps.files.outputs.all_files }}
 
+      # Apply fixed files via pull request review comments
+      - name: Apply any eslint fixed files as Pull Request review comments
+        if: github.ref != 'refs/heads/master' && steps.changes.outputs.backend == 'true'
```

Comment:
Oh I forgot to mention, you should add `github.ref != 'refs/heads/master'` to every step.

Reply:
@joshuarli I'm keeping the same logic here where we run lint on master in Travis, but this step should only run on PRs as we don't want (and can't) commit to master